src/bessel.jl

```diff
@@ -758,3 +758,41 @@ External links: [DLMF](https://dlmf.nist.gov/10.2.6), [Wikipedia](https://en.wik
 See also: [`hankelh2(nu,x)`](@ref SpecialFunctions.hankelh2)
 """
 hankelh2x(nu, z) = besselhx(nu, 2, z)
+
+    
+ # In future just use `fastabs` from Base.Math
+ # https://github.com/JuliaLang/julia/blob/93fb785831dcfcc442f82fab8746f0244c5274ae/base/special/trig.jl#L1057
+function fastabs(x)
```

Comment:
You can do this the other way:
```jl
if isdefined(Base.Math, :fastabs)
    import Base.Math: fastabs
else
    fastabs(x) = abs(x)
    fastabs(x::Complex) = abs(real(x)) + abs(imag(x)) 
end
```

Reply:
This definition should probably go into SpecialFunctions.jl and be used in a few other places in the code (e.g. https://github.com/JuliaMath/SpecialFunctions.jl/blob/9116108ba52d103a05d47d9af1ca6b959a536c30/src/gamma.jl#L477 and https://github.com/JuliaMath/SpecialFunctions.jl/blob/9116108ba52d103a05d47d9af1ca6b959a536c30/src/expint.jl#L181 and https://github.com/JuliaMath/SpecialFunctions.jl/blob/9116108ba52d103a05d47d9af1ca6b959a536c30/src/expint.jl#L181)

Reply:
For `gamma.jl` it seems fine, I could include that.
But the other two are slightly different, aren't they?

Reply:
They are slightly different, but `fastabs` is fine as a replacement in the expint case.